orchagent/fgnhgorch.h

```diff
@@ -95,6 +100,10 @@ class FgNhgOrch : public Orch
     FgPrefixOpCache m_fgPrefixAddCache;
     FgPrefixOpCache m_fgPrefixDelCache;
 
+    // warm reboot support for recovery
+    // < ip_prefix, < ip_next_hop, HashBuckets>>
+    map<string, NextHopIndexMap> m_recoveryMap;
```

Comment:
nit: can we define ```typedef map<string, NextHopIndexMap>WarmBootRecoveryMap; ``` above and have this written as ```WarmBootRecoveryMap m_recoveryMap;``` ?

Reply:
Make the change